app/javascript/packs/application.js

```diff
@@ -7,4 +7,15 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
-console.log('Hello World from Webpacker')
+console.log('You\'re doing great!')
```

Comment:
we should eliminate any `console.log` or `binding.pry` statements prior to pull requesting.